electrumx/server/session.py

```diff
@@ -575,7 +575,7 @@ def add_session(self, session):
                                 'connections until count drops to {:,d}'
                                 .format(self.max_sessions, self.low_watermark))
             loop = asyncio.get_event_loop()
-            loop.call_soon(self._close_servers(['TCP', 'SSL']))
+            asyncio.run_coroutine_threadsafe(self._close_servers(['TCP', 'SSL']), loop)
```

Comment:
I'm not sure why this solves the problem you saw?

Reply:
I'm near-certain run_coroutine_threadsafe is not needed for anything in ElectrumX - superstition?

Reply:
it solves the problem as `call_soon` expects a traditional method callback, while `run_coroutine_threadsafe` expects a coroutine